### ISSUE TITLE
Предложение: разрешить загрузку автатара при регистрации.

### ISSUE THREAD

**Issue by Risgit** [instantsoft/icms2#1188]
Всю плешь уже проели этими аватарами при регистрации.
Может на самом деле разрешить загрузку аватара именно при регистрации?
То есть 
```
if (!$this->cms_user->is_logged && !$this->request->hasInQuery('register')) {
            return $this->cms_template->renderJSON(array(
                'success' => false,
                'error'   => 'auth error'
            ));
        }

```
А потом, по завершении регистрации перемещать аватары в папку пользователя:
То есть, к примеру, id нового юзера 2345.
Генерим папку /upload/023/u2345 и перемещаем /upload/000/u0/5/c/f9bbf6db.jpg , который сформировался при загрузке от гостя в /upload/023/u2345/5/c/f9bbf6db.jpg.
А папку /upload/000/u0 можно очищать по расписанию.

**Comment by vlad7090** [comment]
да, было бы отлично. Также аватарку желательно давать обрезать как в ВК, так как иначе грузят несмотря на размеры файлы по 10 мб.
